Bundler/Processor/ProcessorService/AssetCreator/App.config

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <configSections>
+    <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+      
+      <section name="AssetCreator.UnitySettings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
+      <section name="AssetCreator.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
+    </sectionGroup>
+  </configSections>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+  </startup>
+  <userSettings>
+    
+    <AssetCreator.UnitySettings>
+      <setting name="UnityInstallLocation" serializeAs="String">
+        <value>C:\Program Files\Unity\Editor\</value>
```

Comment:
Is this going to be an issue for folks who use different install locations? Is the fix obvious to them?

Reply:
Added code to dump the settings, check these paths, and give helpful error message if the path doesn't exist.